youtubearchive.py

```diff
@@ -0,0 +1,87 @@
+from dataclasses import dataclass
+import dataclasses
```

Comment:
I would prefer either `from dataclasses import dataclass, asdict`, or just `import dataclasses` with `@dataclasses.dataclass` as the decorator. Personal preference.